index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title> home </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./css/styles.css"/>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <div class="brand">
+                    <h1> My Logo </h1>
+                </div>
+                <div>
+                    <img src="./images/dashicons_menu.png" alt="dashicon menu icon"/>
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <div class="title">
+                <h1> I’m Mariane Glad to see you! </h1>               
+            </div>
+            <div>
+                <article>
+                    I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </article>
```

Comment:
This is a paragraph. Make use of the `<p>` tag instead of the article semantic tag.